movement_primitives/dmp/_dmp_with_final_velocity.py

```diff
@@ -35,6 +35,12 @@ class DMPWithFinalVelocity(WeightParametersMixin, DMPBase):
         Gain for proportional controller of DMP tracking error.
         The domain is [0, execution_time**2/dt].
 
+    alpha_y : float, list with length n_dims, or array with shape (n_dims,), optional (default: 25.0)
```

Comment:
same applies here

Reply:
Addressed in [7e5d15e](https://github.com/dfki-ric/movement_primitives/pull/59/commits/7e5d15ea3cd1587d8f2bbca949f1a8385325a8f8)